Makefile

```diff
@@ -8,18 +10,18 @@ WOODPECKER_CONFIG := $(WOODPECKER_HOME)/config.json
 WOODPECKER_ISSUER := $(WOODPECKER_HOME)/issuer.pem
 WOODPECKER_ISSUER_KEY := $(WOODPECKER_HOME)/issuer.key
 
+WOODPECKER_BARE_REPO := /usr/local/src/ct-woodpecker.git
+WOODPECKER_UPSTREAM := https://github.com/letsencrypt/ct-woodpecker.git
+
 WOODPECKER_CMD := $(BIN_DIR)/ct-woodpecker
 WOODPECKER_DEFAULT_CONFIG := ./util/config.dist.json
 WOODPECKER_DEFAULT_ISSUER := ./test/issuer.pem
 WOODPECKER_DEFAULT_ISSUER_KEY := ./test/issuer.key
 WOODPECKER_SERVICE := ./util/ct-woodpecker.service
+WOODPECKER_POST_RECV_HOOK := ./util/post-receive.git.hook.example
```

Comment:
Let's not call this .example because we're using it for real.

Reply:
Sure, fixed in 259d895